cmd/microcloud/ask.go

```diff
@@ -151,7 +161,12 @@ func (c *initConfig) askRetry(question string, f func() error) error {
 				return err
 			}
 
-			tui.PrintError(err.Error())
+			if errors.As(err, &askRetryWarningError{}) {
```

Comment:
Just a nit for readability:

```suggestion
			var retryWarningErr askRetryWarningError
			if errors.As(err, &retryWarningErr) {
```

Reply:
Fixed.